src/Commands/InstallCommand.php

```diff
@@ -0,0 +1,166 @@
+        }
+
+        if ($this->shouldPublishMigrations) {
+            $this->info('Publishing migration...');
+
+            $this->callSilently("vendor:publish", [
+                '--tag' => "{$this->package->shortName()}-migrations",
+            ]);
+        }
+
+        if ($this->copyServiceProviderInApp) {
+            $this->copyServiceProviderInApp();
+        }
+
+        if ($this->starRepo) {
+            if ($this->confirm('Would you like to star our repo on GitHub?')) {
+                $repoUrl = "https://github.com/{$this->starRepo}";
+
+                if (PHP_OS_FAMILY == 'Darwin') {
+                    exec("open {$repoUrl}");
+                }
+                if (PHP_OS_FAMILY == 'Windows') {
+                    exec("start {$repoUrl}");
+                }
+                if (PHP_OS_FAMILY == 'Linux') {
+                    exec("xdg-open {$repoUrl}");
+                }
+            }
+        }
+
+        $this->info("{$this->package->shortName()} has been installed!");
+
+        if ($this->endWith) {
+            ($this->endWith)($this);
+        }
+    }
+
+    public function publishConfigFile(): self
+    {
+        $this->shouldPublishConfigFile = true;
+
+        return $this;
+    }
+
+    public function publishMigrations(): self
+    {
+        $this->shouldPublishMigrations = true;
+
+        return $this;
+    }
+
+    public function copyAndRegisterServiceProviderInApp(): self
+    {
+        $this->copyServiceProviderInApp = true;
+
+        return $this;
+    }
+
+    public function askToStarRepoOnGitHub($vendorSlashRepoName): self
+    {
+        $this->starRepo = $vendorSlashRepoName;
+
+        return $this;
+    }
+
+    public function startWith($callable): self
+    {
+        $this->startWith = $callable;
+
+        return $this;
+    }
+
+    public function endWith($callable): self
+    {
+        $this->endWith = $callable;
+
+        return $this;
+    }
+
+    protected function copyServiceProviderInApp(): self
+    {
+        dump('copying');
+        $providerName = $this->package->publishableProviderName;
+
+        if (! $providerName) {
+            dump('no provider name');
+
+            return $this;
+        }
+
+        $this->callSilent('vendor:publish', ['--tag' => $this->package->shortName() . '-provider']);
+
+        $namespace = Str::replaceLast('\\', '', $this->laravel->getNamespace());
+
+        $appConfig = file_get_contents(config_path('app.php'));
+
+        $class = '\\Providers\\' . $providerName . '::class';
+
+        if (Str::contains($appConfig, $namespace . $class)) {
+            dump('already contains');
+
+            return $this;
+        }
+
+        dump('updating');
+        file_put_contents(config_path('app.php'), str_replace(
+            "Illuminate\\View\ViewServiceProvider::class," . PHP_EOL,
+            "Illuminate\\View\ViewServiceProvider::class," . PHP_EOL . "        {$namespace}\Providers\\" . $providerName . "::class," . PHP_EOL,
```

Comment:
```suggestion
            "Illuminate\\View\ViewServiceProvider::class,",
            "Illuminate\\View\ViewServiceProvider::class," . PHP_EOL . "        {$namespace}\Providers\\" . $providerName . "::class,",
```

This should fix the Windows Problem

Reply:
The same change was made here: https://github.com/laravel/jetstream/pull/55/files

Reply:
Thanks, made the change, tests are running 🤞